packages/@o3r/components/builders/metadata-check/helpers/config-metadata-comparison.helper.ts

```diff
@@ -0,0 +1,49 @@
+import type { ComponentConfigOutput } from '@o3r/components';
```

Comment:
I know we have integration tests but do you think its worth adding Unit Tests for this pure functions?